CONTRIBUTING.md

```diff
@@ -105,20 +105,33 @@ For the `_info_` function, you will need to define `features` for your
 2. [Relation Extraction (RE)](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/kb.py)
 3. [Event Extraction](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/kb.py)
 4. [Named entity disambiguation/canonicalization/normalization (NED)](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/kb.py)
-5. [Question-Answering](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/qa.py)
-6. [Entailment](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/entailment.py)
-7. [Translation](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text_to_text.py)
-8. [Summarization](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text_to_text.py)
-9. [Paraphrasing](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text_to_text.py)
-10. [Sentence/Phrase/Text classification](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text.py)
-11. [Pair Labels](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/pairs.py)
+5. [Co-reference resolution](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/kb.py)
+6. [Question-Answering](https://github.com/bigscience-workshop/biomedical/blob/aster/schemas/qa.py)
+7. [Entailment](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/entailment.py)
+8. [Translation](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text_to_text.py)
+9. [Summarization](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text_to_text.py)
+10. [Paraphrasing](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text_to_text.py)
+11. [Sentence/Phrase/Text classification](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/text.py)
+12. [Pair Labels](https://github.com/bigscience-workshop/biomedical/blob/master/schemas/pairs.py)
 
 You will use this schema in the `_generate_examples` return value.
 
 Please read the [Task Schemas](#task_schemas.md) to understand how each key should behave.
 
 Populate the information in the dataset according to this schema; some fields may be empty.
 
+To enable quality control, please add the following line in your file before the class definition:
+```python
+_SUPPORTED_TASKS = ["your_task_name_here"]
+```
+For ease, please refer to the following keywords, given your NLP task:
+1. **NER, RE, Event Extraction, NED/ Coreference resolution**: `"kb"`
```

Comment:
I suggest we break out the tasks here b/c some datasets will support one or more of these tasks so `_SUPPORTED_TASKS` would have things like `ner`, `re`, `ee`, `ned`, `coref`, ....

Reply:
sounds fine to me; this was something I made a pragmatic judgement call to make it easier to check schemas but I agree this is way better

I'd like to merge @leonweber's test here too cause he had a nice strategy where you specify the dataset name whereas mine just searches in the folder; it would be good to integrate these two ideas as we can either do a quick search through all the datasets (For later) OR have the ability to check a specific dataset + pass any necessary flags with argparse